components/Footer.tsx

```diff
@@ -111,7 +111,7 @@ export const FooterImpl: React.FC = () => {
         )}
         <a href="https://drive.google.com/file/d/1966dZX4w8KF84IpvKdBds5ILVYiu3N-k/view?usp=sharing" 
           className={styles.readme}>
-          <FaReadme/>
+          <FaZhihu/>
```

Comment:
Issue with this line - I imported FaReadme icon but did not use it. So I will not import it and try again with testing this update.